pkg/openapi/server.spec.yaml

```
@@ -322,12 +459,28 @@ components:
       required: true
       schema:
         $ref: '#/components/schemas/kubernetesNameParameter'
+    regionTypeParameter:
+      name: regionType
+      in: query
+      description: |-
+        The type of region we are asking for.
+      required: true
+      schema:
+        $ref: '#/components/schemas/regionTypeParameter'
   schemas:
     kubernetesNameParameter:
       description: A Kubernetes name. Must be a valid DNS containing only lower case characters, numbers or hyphens, start and end with a character or number, and be at most 63 characters in length.
       type: string
       minLength: 1
       maxLength: 63
+    regionTypeParameter:
+      description: |-
+        The region type, physical means a full Kubernetes cluster deployment, virtual is a virtual
```

```suggestion
        The region type, 'physical' means a full Kubernetes cluster deployment, 'virtual' is a virtual
```
(quote the \<var\>s)